app/pages/project/index.spec.js

```diff
@@ -49,45 +49,100 @@ const resources = {
   ]
 };
 
-const project = apiClient.type('projects').create({
-  id: 'a',
-  display_name: 'A test project',
-  experimental_tools: [],
-  get(type) {
-    return Promise.resolve(resources[type]);
-  },
-  links: {
-    active_workflows: ['1', '2', '3', '4', '5'],
-    avatar: { id: '1' },
-    background: { id: '1' },
-    owner: { id: '1' }
+function mockPanoptesResource(type, options) {
+  const resource = apiClient.type(type).create(options);
+  if (!resource.save.restore) {
+    sinon.stub(resource, 'save', () => Promise.resolve(resource));
+    sinon.stub(resource, 'get', resourceType => Promise.resolve(resources[resourceType]));
```

Comment:
Not a blocker but `resources` here is a little confusing when reading for the first time. It looks like it's a prebuilt instance with known resource types and this get lookup will just fetch one of them, maybe rename to `factoryResources` or some such to know it's a known prebuilt resource lookup.

Reply:
It should probably be named `RESOURCES`, in keeping with constants in other files.